Formula/o/openjdk@8.rb

```diff
@@ -1,9 +1,9 @@
 class OpenjdkAT8 < Formula
   desc "Development kit for the Java programming language"
   homepage "https://openjdk.java.net/"
-  url "https://github.com/openjdk/jdk8u/archive/refs/tags/jdk8u412-ga.tar.gz"
-  version "1.8.0-412"
-  sha256 "f63bb60fbc6e798b0bbf5b6477765c66740a2b3cceab2d6713be879057f1c99b"
+  url "https://github.com/openjdk/jdk8u/archive/refs/tags/jdk8u422-b05.tar.gz"
```

Comment:
Judging from tags on https://github.com/openjdk/jdk8u/tags
this seems to be beta 5 (`b05`), not a generally available (`ga`) release.
<img width="344" alt="image" src="https://github.com/user-attachments/assets/64cec4cb-0f31-44da-83fc-0d9480f53111">

> openjdk@8: Formula version newer than livecheck
The formula version (1.8.0-422) is newer than the version from `brew livecheck` (1.8.0+412).

Reply:
Couldn't they add the `ga` tag to the same commit instead of releasing it as `b05`?
https://mail.openjdk.org/pipermail/jdk8u-dev/2024-July/018904.html

Reply:
New `ga` tag was also suggested on that mailing list: https://mail.openjdk.org/pipermail/jdk8u-dev/2024-July/018909.html
> It would be good if the associated -ga tag could be pushed.

Reply:
Yes, I understand this, but up until now, they have neither responded nor taken any action to add the `ga` tag. 

So we have two choice: First, we can put this PR on hold and wait until they add the `ga` tag (which might not happen until the next release of `1.8.0-432`), and then continue building `1.8.0-422`. The other option is to go ahead and build `1.8.0-422` based on the `b05` tag, and if they add the `ga` tag later, we can rebuild based on the `ga` tag.

[Azul Zulu](https://www.azul.com/downloads/?version=java-8-lts&package=jdk#zulu) and [Eclipse Temurin (adoptium)](https://adoptium.net/zh-CN/temurin/releases/?version=8) seem to choose the former.  @stefanb

Reply:
> Judging from tags on https://github.com/openjdk/jdk8u/tags this seems to be beta 5 (`b05`), not a generally available (`ga`) release. <img alt="image" width="344" src="https://private-user-images.githubusercontent.com/319826/351578439-64cec4cb-0f31-44da-83fc-0d9480f53111.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4MDM2NDksIm5iZiI6MTcyMTgwMzM0OSwicGF0aCI6Ii8zMTk4MjYvMzUxNTc4NDM5LTY0Y2VjNGNiLTBmMzEtNDRkYS04M2ZjLTBkOTQ4MGY1MzExMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyNFQwNjQyMjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZTk2MDU4NjUwMmZlMjU3YjdiNjQ4NGMzNjc1ZTFjMzQ1OTg3N2FiY2E3N2RlYjVhMmNmZDIyZTdmN2M1M2VkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.fbMnl_BIgPaWbtRGos378m80WostEZpo4aXcThQQzMw">
> 
> > openjdk@8: Formula version newer than livecheck
> > The formula version (1.8.0-422) is newer than the version from `brew livecheck` (1.8.0+412).

I think `b` here represents `build` rather than `beta`. `b05` means that it is the fifth time.